### ISSUE TITLE
HOWTO

### ISSUE THREAD

**Issue by alexeykudinkin** [intellij-rust/intellij-rust#505]
As we move forward, more and more features (primarily language ones for now) get alive, though we still do not have a consistent way to notify all the rustaceans about what we're up to.

So my suggestion is to create `HOWTO.md` and maintain it up to par with our current state of the art, and maybe even demoing some of our features with a help of short gifs.

**Comment by Furyhunter** [comment]
Why not make a Jekyll site on gh-pages instead?

**Comment by jajakobyly** [comment]
> Why not make a Jekyll site on gh-pages instead?

I can make one if nobody else wants, though I don't know whether it makes sense to do this on current stage of the project.

**Comment by alexeykudinkin** [comment]
@Furyhunter it'd be nice to have it (at least partially) embedded into `README.md`.

**Comment by alexeykudinkin** [comment]
@jajakobyly it definitely has, since most of the people wouldn't dare to try until sure, what they can see under the hood. 

And we already have some stuff to show.

**Comment by matklad** [comment]
We have a page with similar intents here: https://github.com/intellij-rust/intellij-rust/blob/master/doc/Features.md. Adding some gifs there would be very nice. And I am for gh-pages, they should be pretty easy to maintain.

**Comment by jajakobyly** [comment]
So I'll create something interesting, after #504 :wink:

**Comment by jajakobyly** [comment]
The website is being developed here: https://github.com/intellij-rust/intellij-rust.github.io

https://intellij-rust.github.io/

**Comment by jajakobyly** [comment]
@matklad this issue is ready for garbage collector ;)

**Comment by matklad** [comment]
Yeah, let's close this one and open more specific for 

> it'd be nice to have it (at least partially) embedded into README.md.

and

> demoing some of our features with a help of short gifs.

**Comment by matklad** [comment]
#522 #521
